packages/ui-components/src/components/form/RadioGroup.stories.ts

```diff
@@ -0,0 +1,73 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
+import RadioGroup from './RadioGroup.vue'
+import { ChartBarIcon, WrenchIcon, WalletIcon } from '@heroicons/vue/24/outline'
+import type { ConcreteComponent } from 'vue'
+
+type RadioGroupStoryType = StoryObj<{
+  'update:modelValue'?: (val: string) => void
+  modelValue: string
+  options: Array<{
+    value: string
+    title: string
+    introduction: string
+    icon: ConcreteComponent
+    help?: string
+  }>
+}>
+
+export default {
+  component: RadioGroup,
+  argTypes: {
+    'update:modelValue': {
+      action: 'update:modelValue',
+      type: 'function'
+    }
+  },
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'A customizable radio group component with titles, icons, and introductions for each option.'
+      }
+    }
+  }
+} as Meta
+
+export const Default: RadioGroupStoryType = {
+  render: (args) => ({
+    components: { RadioGroup },
+    setup() {
+      return { args }
+    },
+    template: `
+      <div>
+        <RadioGroup v-bind="args" @update:modelValue="args['update:modelValue']" />
```

Comment:
There's no two-way binding between the Storybook arg and the model value. If I change the value by clicking on the button, I won't see the new value pop up in the controls panel. I suggest using the `useStorybookVmodel` composable I created to simplify doing so